pkg/operator/starter.go

```diff
@@ -63,6 +66,12 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		return err
 	}
 
+	mapiClient, err := mapiclientset.NewForConfig(controllerContext.KubeConfig)
+	if err != nil {
+		return err
+	}
+	machineSharedInformer := machineinformersv1beta1.NewSharedInformerFactoryWithOptions(mapiClient, 10*time.Minute, machineinformersv1beta1.WithNamespace("openshift-machine-api"))
```

Comment:
What is the `10*time.Minute`? Resync period? Might be worth converting to a const for reuse (and so we know what it is in the future)